docs/projects/factorial/_posts/2018-12-19-python.md

```diff
@@ -0,0 +1,163 @@
+
+---
+title: Factorial in Python
+
+layout: default
+
+last-modified: 2019-10-26
+
+featured-image:
+
+tags: [python, factorial]
+
+authors:[ underscoreanuj ]
```

Comment:
Make sure you add yourself to the author file: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml